symfony/src/Service/DeceasedNotificationService.php

```diff
@@ -30,25 +29,22 @@ class DeceasedNotificationService extends EmailNotificationService
     protected $deceasedStatus;
 
     public function __construct(
-        EntityManagerInterface $em,
         ManagerRegistry $managerRegistry,
         ParticipantSummaryService $participantSummaryService,
         LoggerService $loggerService,
         EnvironmentService $env,
         ParameterBagInterface $params,
-        Environment $twig,
-        SiteRepository $siteRepository,
-        DeceasedLogRepository $deceasedLogRepository
+        Environment $twig
     ) {
-        $this->em = $em;
+        $this->em = $managerRegistry->getManager();
         $this->managerRegistry = $managerRegistry;
         $this->participantSummaryService = $participantSummaryService;
         $this->loggerService = $loggerService;
         $this->env = $env;
         $this->params = $params;
         $this->twig = $twig;
-        $this->siteRepository = $siteRepository;
-        $this->logRepository = $deceasedLogRepository;
+        $this->siteRepository = $this->em->getRepository(Site::class);
+        $this->logRepository = $this->em->getRepository(DeceasedLog::class);
```

Comment:
Thanks for cleaning this up. I just noticed that this class redefines the same class variables as in the parent EmailNotificationService class. Can you remove the ones that aren't set here? You should only need to include the ones that aren't in the parent class. Or are overriding the default value.